package.json

```diff
@@ -21,11 +23,21 @@
     "mysql2": "^1.6.5"
   },
   "devDependencies": {
+    "@types/mysql2": "github:types/mysql2",
```

Comment:
Add type definitions for the `mysql2` package. If you're using VS Code, this will allow the editor to show us definitions for the functions of this package when hovering over objects/functions that we import from that package. It will also help autocomplete stuff when using that package.